members/CusB02d45e321534531912.yaml

```diff
@@ -0,0 +1,152 @@
+bioguide: CusB02d45e321534531912
+contact_form:
+  method: post
+  action: ""
+  useChrome: true
+  steps:
+    - visit: "https://eplanning.blm.gov/epl-front-office/eplanning/comments/commentSubmission.do?commentPeriodId=70305"
+    - wait:
+      - value: 2
+    - find:
+        - selector: input[id='commentSubComment.commentTitle']
+    - fill_in:
+        - name: Subject
+          selector: input[id='commentSubComment.commentTitle']
+          value: $SUBJECT
+          required: false
+    - iframe:
+        - name: message
+          selector: iframe[class='cke_wysiwyg_frame cke_reset']
+    - find:
+        - selector: body[class='cke_editable cke_editable_themed cke_contents_ltr cke_show_borders']
+    - fill_in:
+        - name: message
+          selector: body[class='cke_editable cke_editable_themed cke_contents_ltr cke_show_borders']
+          value: $MESSAGE
+          required: true
+    - iframe:
+        - back: true
+    - click_on:
+        - selector: input[type='submit'][value='Next']
+    - wait:
+        - value: 1
+    - find:
+        - selector: input[id*='.firstName']
+    - fill_in:
+        - name: First Name
+          selector: input[id*='.firstName']
+          value: $NAME_FIRST
+          required: true
+        - name: Last Name
+          selector: input[id*='lastName']
+          value: $NAME_LAST
+          required: true
+        - name: City
+          selector: input[id*='.city']
+          value: $ADDRESS_CITY
+          required: true
+        - name: zip5
+          selector: input[id*='.zip']
+          value: $ADDRESS_ZIP5
+          required: true
+        - name: email
+          selector: input[id*='.emailAddress']
+          value: $EMAIL
+          required: false
+        - name: phone
+          selector: input[id*='.dayPhone']
+          value: $PHONE
+          required: false
+    - select:
+        - name: Prefix
+          selector: select[id*='.title']
+          value: $NAME_PREFIIX
```

Comment:
Should be `$NAME_PREFIX`  - typo with extra I in there

Reply:
ohhhh, thanks for flagging. Let me fix that